spdx-utils/src/main/kotlin/SpdxConstants.kt

```diff
@@ -35,6 +35,11 @@ object SpdxConstants {
      */
     const val NOASSERTION = "NOASSERTION"
 
+    /**
+     * The URL that points to list of SPDX licenses on https://spdx.org.
+     */
+    const val SPDX_BASE_URL = "https://spdx.org/licenses/"
```

Comment:
"BASE" is unclear WRT "base" of *what*? I'd prefer to call this `SPDX_LICENSE_LIST_URL`.

Reply:
Renamed to `SPDX_LICENSE_LIST_URL`.